fiddler/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield
+
+
+@pytest.fixture
+def instance():
+    return {}
```

Comment:
Since the [tests](https://github.com/DataDog/integrations-extras/pull/1383/files#diff-19fdaaff107e9f79e47188994940f12e48f9068c76098af73b8eae73e05cc5adR11) are using `dd_run_check` to run a check instance, the instance configuration needs to be defined here:
```suggestion
    return {"url": "https://demo.trial.fiddler.ai", "fiddler_api_key": "apikey", "organization": "demo"}
```

Reply:
Done